src/pages/ProductListing.js

```diff
@@ -0,0 +1,60 @@
+import { Navbar } from "../components/Navbar"
+import { Footer } from "../components/Footer"
+import "../styles/pages/productlist.css"
+import { FiltersBar } from "../components/FiltersBar"
+import { ProductCard } from "../components/ProductCard"
+import {useState,useEffect} from "react";
+import axios from "axios"
+import { useProduct } from "../context/product-context"
+import {filterProducts} from "../utils/filterProducts"
+
+
+
+export const ProductListing = ()=>{
+    const [loader, setLoader] = useState(true);
+    const [products, setProducts] = useState([{}]);
+    const {state} = useProduct();
+
+    useEffect(() => {
```

Comment:
create a folder services and you can put the API fetch calls over there for cleaner code

Reply:
Done, Thanks for Reviewing my PR